spec-main/extensions-spec.ts

```diff
@@ -296,7 +296,7 @@ ifdescribe(process.electronBinding('features').isExtensionsEnabled())('chrome ex
       const customSession = session.fromPartition(`persist:${require('uuid').v4()}`);
       customSession.loadExtension(path.join(fixtures, 'extensions', 'devtools-extension'));
       const w = new BrowserWindow({ show: true, webPreferences: { session: customSession, nodeIntegration: true } });
-      await w.loadURL('data:text/html,hello');
+      await w.loadURL(url);
```

Comment:
hm, reading through the linked patch, i'm surprised this change is needed. I don't think `data:` is a privileged page?

Reply:
Yeah I was confused by it too but they also check for origin of the execution context https://source.chromium.org/chromium/chromium/src/+/master:third_party/devtools-frontend/src/front_end/extensions/ExtensionServer.js;l=980

For `about` and `data` the origin turns as `://`